docs/admin_api/user_admin_api.md

```diff
@@ -443,8 +443,9 @@ The following fields are returned in the JSON response body:
 - `joined_rooms` - An array of `room_id`.
 - `total` - Number of rooms.
 
+## User media
 
-## List media of a user
+### List media of a user
```

Comment:
this should be "uploaded by" as well, but I don't know if we have other links to it